.github/workflows/pipeline-deploy-to-internal.yml

```diff
@@ -0,0 +1,42 @@
+name: "[Pipeline] Deploy to Internal"
+
+on:
+    workflow_call:
+
+jobs:
+
+    build-internal-aab:
+        uses: ./.github/workflows/reusable-build-apk.yml
+        secrets: inherit
+        with:
+            build-environment: internal
+
+    run-app-sweep:
+        uses: ./.github/workflows/reusable-app-sweep.yml
+        with:
+            upload-artifact: ${{ needs.build-internal-aab.outputs.build-artifact }}
+            mapping-file: ${{ needs.build-internal-aab.outputs.optional-mapping-file }}
+        secrets: inherit
+        needs:
+            - build-internal-aab
+
+    deploy-internal-build:
+        uses: ./.github/workflows/reusable-promote-artifact.yml
+        with:
+            deployment-track: internal
+            upload-artifact: ${{ needs.build-internal-aab.outputs.build-artifact }}
+            mapping-file: ${{ needs.build-internal-aab.outputs.optional-mapping-file }}
+        needs:
+            - build-internal-aab
+            - run-app-sweep
+
+    deploy-alpha-build:
```

Comment:
Hm, do we always want to deploy simultaneously to both Internal and Alpha? Looking at the lists of both - I'm not sure what's the distinction (and hence use of Alpha) currently.

Reply:
This won't deploy them simultaneously. Since Alpha is a protected environment what it will do is trigger the request to deploy to alpha. But the step isn't run until it's approved.